modules/azure/databricks_permissions_user_based/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "workspace_url" {
+  type        = string
+  description = "The url of the databricks workspace."
+}
+
+variable "users" {
+  type        = list(string)
+  description = "A list of users in the databricks workspace."
+  default     = []
```

Comment:
@jvanenckevort Is it possible for there to be no users or admins?

Then we would only have a `databricks_token` and `databricks_permissions` resource to be created